client/components/loading-placeholder/index.jsx

```diff
@@ -0,0 +1,35 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import noop from 'lodash/utility/noop';
+import page from 'page';
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import HeaderCake from 'components/header-cake';
+import Main from 'components/main';
+
+const LoadingPlaceholder = React.createClass( {
+	propTypes: {
+		className: React.PropTypes.string
+	},
+
+	goBack() {
+		page.back( '/' );
+	},
+
+	render() {
+		return (
+			<Main className={ classnames( 'loading-placeholder', this.props.className ) }>
+				<HeaderCake className="loading-placeholder__header" onClick={ this.goBack } />
```

Comment:
Shouldn't we also display the page title here? I mean this is something that doesn't require data being fetched from the API most of the time. Or maybe this just adds too much complexity?